server/api.js

```diff
@@ -0,0 +1,8 @@
+const api = require('express').Router();
+
+api.use('/auth', require('./auth'));
+
+// No routes matched? 404.
```

Comment:
Deferring to @parky22 on this one. I personally like having the option of being able to throw in more routes super easily if/when the time comes...